lib/erlef/grant_mail.ex

```diff
@@ -9,22 +9,25 @@ defmodule Erlef.GrantMail do
 
   @spec submission(%Erlef.GrantProposal{}) :: %Swoosh.Email{}
   def submission(proposal) do
+    dt_str = DateTime.to_string(DateTime.utc_now())
```

Comment:
Nit: I think this would clutter the email subject. For an email subject, I think we can leave time and microseconds out :) A `year/month/day` should suffice.

```
# Current
iex(1)> dt_str = DateTime.to_string(DateTime.utc_now())                                      
"2019-09-22 15:23:53.341236Z"

# Proposed
# Source: https://stackoverflow.com/a/47791946/3163631
iex(2)> date =  Date.utc_today
~D[2019-09-22]
iex(3)> :io_lib.format("~4..0B/~2..0B/~2..0B", [date.year, date.month, date.day]) |> IO.iodata_to_binary
"2019/09/22"
```

Reply:
Yeah, if we only use want the year, month, and day we can just use `Date.utc_today/0`, no need to format it I think. Actually the call to DateTime.to_string/1 is a bit pointless here since elixir will stringify for us.  

 The purpose is simply to make the subject a bit more unique since we currently are not backed by a database and thus have no unique id to append. The thought behind the time is that the same entity could submit several grant proposals in a day, the micro-seconds might be a bit much though :)

Reply:
Thanks, both for your comments this is very valid feedback. I agree microseconds is not really necessary but it is a good compromise for now.

Reply:
Sounds good, I'll remove the ms.